RZAssemblage/RZAssemblage.h

```diff
@@ -6,14 +6,124 @@
 //  Copyright (c) 2015 Raizlabs. All rights reserved.
 //
 
-#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
 
-//! Project version number for RZAssemblage.
 FOUNDATION_EXPORT double RZAssemblageVersionNumber;
-
-//! Project version string for RZAssemblage.
 FOUNDATION_EXPORT const unsigned char RZAssemblageVersionString[];
 
-// In this header, you should import all the public headers of your framework using statements like #import <RZAssemblage/PublicHeader.h>
+#import "RZAssemblageChangeSet.h"
+
+OBJC_EXTERN NSString *const RZAssemblageUpdateKey;
+
+@class NSFetchedResultsController;
+
+typedef NS_OPTIONS(NSUInteger, RZAssemblageEnumerationOptions) {
+    RZAssemblageEnumerationNoOptions = 0,
```

Comment:
Probably don't need to specify a `NoOptions` since `kNilOptions` exists. Or, point `RZAssemblageNilOptions = kNilOptions`